Tests/test_image_access.py

```diff
@@ -415,7 +415,9 @@ def test_embeddable(self) -> None:
 
 int main(int argc, char* argv[])
 {
-    char *home = "%s";
+    char *home = \""""
+                + sys.prefix.replace("\\", "\\\\")
```

Comment:
Can we put this in a variable outside the string, then make this an f-string?

Reply:
Ok, I've pushed a commit. It did require escaping brackets though.